[Bug] - rz-ghidra build error on latest

Rizin Version (built on `fedora:latest`)
```
# rizin -v
rizin 0.5.0 @ linux-x86-64
commit: 87c850e43100b46c8641ed24e463293f30bbd271
```

When building rz-ghidra (pulling from `master`) the following error is produced
```c
/rz-ghidra/src/analysis_ghidra.cpp: In function ‘int archinfo(RzAnalysis*, int)’:
/rz-ghidra/src/analysis_ghidra.cpp:39:21: error: ‘RZ_ANALYSIS_ARCHINFO_ALIGN’ was not declared in this scope; did you mean ‘RZ_ANALYSIS_ARCHINFO_TEXT_ALIGN’?
   39 |         if(query == RZ_ANALYSIS_ARCHINFO_ALIGN)
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~
      |                     RZ_ANALYSIS_ARCHINFO_TEXT_ALIGN
/rz-ghidra/src/analysis_ghidra.cpp: At global scope:
/rz-ghidra/src/analysis_ghidra.cpp:3290:27: error: invalid conversion from ‘int (*)(RzAnalysis*, int)’ {aka ‘int (*)(rz_analysis_t*, int)’} to ‘int (*)(RzAnalysis*, RzAnalysisInfoType)’ {aka ‘int (*)(rz_analysis_t*, rz_analysis_arch_info_type_t)’} [-fpermissive]
 3290 |         /* .archinfo = */ &archinfo,
      |                           ^~~~~~~~~
      |                           |
      |                           int (*)(RzAnalysis*, int) {aka int (*)(rz_analysis_t*, int)}
/rz-ghidra/src/analysis_ghidra.cpp: In function ‘int archinfo(RzAnalysis*, int)’:
/rz-ghidra/src/analysis_ghidra.cpp:43:1: warning: control reaches end of non-void function [-Wreturn-type]
   43 | }
      | ^
make[2]: *** [CMakeFiles/analysis_ghidra.dir/build.make:118: CMakeFiles/analysis_ghidra.dir/src/analysis_ghidra.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:198: CMakeFiles/analysis_ghidra.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 99%] Linking CXX shared library asm_ghidra.so
[ 99%] Built target asm_ghidra
make: *** [Makefile:136: all] Error 2
```

The issue can be replicated using the following Dockerfile
```Dockerfile
FROM fedora:latest

RUN dnf install -y python3-devel \
                 python3-pip \
                 cmake \
                 gcc-c++ \
                 gcc \
                 meson \
                 ninja-build \
                 git \
                 zlib-devel \
                 openssl \
                 openssl-devel \
                 python3 \
                 autoconf \
                 automake \
                 libtool \
                 python3-pkgconfig \
                 m4 \
                 upx \
                 doxygen \
                 graphviz

RUN git clone https://github.com/rizinorg/rizin.git && \
 cd rizin && \
 meson build && \
 ninja -C build && \
 ninja -C build install 
 
RUN git clone https://github.com/rizinorg/rz-ghidra.git && \
 cd rz-ghidra && \
 git submodule init && \
 git submodule update && \
 mkdir build && cd build && \
 cmake -DCMAKE_INSTALL_PREFIX=~/.local .. && \
 make -j$(nproc) && \
 make install 
 
```

--- a/src/analysis_ghidra.cpp
+++ b/src/analysis_ghidra.cpp
@@ -15,7 +15,7 @@
 
 static SleighAsm sanalysis;
 
-static int archinfo(RzAnalysis *analysis, int query)
+static int archinfo(RzAnalysis *analysis, RzAnalysisInfoType query)
 {
 	// This is to check if RzCore plugin set cpu properly.
 	if(!analysis->cpu)
@@ -36,7 +36,7 @@ static int archinfo(RzAnalysis *analysis, int query)
 		return -1;
 	}
 
-	if(query == RZ_ANALYSIS_ARCHINFO_ALIGN)
+	if(query == RZ_ANALYSIS_ARCHINFO_TEXT_ALIGN)
 		return sanalysis.alignment;
 	else
 		return -1;

--- a/test/db/extras/analysis_ghidra
+++ b/test/db/extras/analysis_ghidra
@@ -81,95 +81,95 @@ e asm.arch
 e asm.cpu
 e asm.bits
 wx 89e1
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx a168a00408
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx a368a00408
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 8945f0
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 8b4510
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 0f4c0d03000000
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx cd80
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 55
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 6a00
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 5d
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 83f853
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 85c0
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 83c410
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 81ec88100000
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx f7ea
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx f7f2
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx d1e8
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx d1e0
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx d1f8
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 0b25f0ff0000
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 2325f0ff0000
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 3325f0ff0000
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 8703
-pdi 1
+pdq 1
 ao | grep type
 EOF
 RUN
@@ -187,11 +187,11 @@ CMDS=<<EOF
 e asm.cpu=AARCH64:LE:64:v8A:gcc
 e asm.arch=ghidra
 wx 298947f9
-pdi 1
+pdq 1
 ao | grep type
 ?e ----
 wx 690200f9 @ 0x200
-pdi 1 @ 0x200
+pdq 1 @ 0x200
 ao @ 0x200| grep type
 EOF
 RUN

--- a/test/db/extras/asm_ghidra
+++ b/test/db/extras/asm_ghidra
@@ -30,7 +30,7 @@ EOF
 CMDS=<<EOF
 pdga
 s main
-pdi 24
+pdq 24
 EOF
 RUN
 
@@ -86,7 +86,7 @@ EOF
 CMDS=<<EOF
 pdga
 s main
-pdi 44
+pdq 44
 EOF
 RUN
 
@@ -115,7 +115,7 @@ EXPECT=<<EOF
 EOF
 CMDS=<<EOF
 pdga
-pdi 18
+pdq 18
 EOF
 RUN
 
@@ -177,7 +177,7 @@ EOF
 CMDS=<<EOF
 pdga
 s main
-pdi 50
+pdq 50
 EOF
 RUN
 
@@ -239,7 +239,7 @@ EXPECT=<<EOF
 EOF
 CMDS=<<EOF
 pdga
-pdi 50
+pdq 50
 EOF
 RUN
 
@@ -262,7 +262,7 @@ EXPECT=<<EOF
 EOF
 CMDS=<<EOF
 pdga
-pdi 12
+pdq 12
 EOF
 RUN
 
@@ -323,7 +323,7 @@ EXPECT=<<EOF
 EOF
 CMDS=<<EOF
 pdga
-pdi 50
+pdq 50
 EOF
 RUN
 
@@ -384,7 +384,7 @@ EXPECT=<<EOF
 EOF
 CMDS=<<EOF
 pdga
-pdi 50
+pdq 50
 EOF
 RUN
 
@@ -448,7 +448,7 @@ EXPECT=<<EOF
 EOF
 CMDS=<<EOF
 pdga
-pdi 50
+pdq 50
 EOF
 RUN
 
@@ -509,7 +509,7 @@ EXPECT=<<EOF
 EOF
 CMDS=<<EOF
 pdga
-pdi 50
+pdq 50
 EOF
 RUN
 
@@ -570,7 +570,7 @@ EXPECT=<<EOF
 EOF
 CMDS=<<EOF
 pdga
-pdi 50
+pdq 50
 EOF
 RUN
 
@@ -632,7 +632,7 @@ EOF
 CMDS=<<EOF
 pdga
 s main
-pdi 50
+pdq 50
 EOF
 RUN
 
@@ -693,7 +693,7 @@ EXPECT=<<EOF
 EOF
 CMDS=<<EOF
 pdga
-pdi 50
+pdq 50
 EOF
 RUN
 
@@ -727,7 +727,7 @@ e asm.cpu=v850
 e asm.bits=32
 pdgss
 wx 8100e009ba154007b8a8042a400e000184ff88990c0a400faba80c0a400faca8
-pdi 10
+pdq 10
 EOF
 EXPECT=<<EOF
 V850:LE:32:default

--- a/test/db/extras/ghidra
+++ b/test/db/extras/ghidra
@@ -1775,7 +1775,7 @@ void sym.Aeropause(Bright *bright, int32_t argc, char **argv)
         bright->ambassador = AMBASSADOR_PURE;
     } else {
         (bright->window).sunlight = bright->morning->saved_argv[1];
-        iVar2 = sym.imp.strcmp((bright->window).sunlight, 0x804a05c);
+        iVar2 = sym.imp.strcmp((bright->window).sunlight, data.0804a05c);
         if (iVar2 == 0) {
             bright->ambassador = AMBASSADOR_REASON;
         } else {
@@ -1935,7 +1935,7 @@ void sym.Aeropause(BrightPtr bright, Bright *argc, char **argv)
         bright->ambassador = AMBASSADOR_PURE;
     } else {
         (bright->window).sunlight = bright->morning->saved_argv[1];
-        iVar2 = sym.imp.strcmp((bright->window).sunlight, 0x804a05c);
+        iVar2 = sym.imp.strcmp((bright->window).sunlight, data.0804a05c);
         if (iVar2 == 0) {
             bright->ambassador = AMBASSADOR_REASON;
         } else {
@@ -1995,7 +1995,7 @@ void sym.Aeropause(BrightTypedefd *bright, int32_t argc, char **argv)
         bright->ambassador = AMBASSADOR_PURE;
     } else {
         (bright->window).sunlight = bright->morning->saved_argv[1];
-        iVar2 = sym.imp.strcmp((bright->window).sunlight, 0x804a05c);
+        iVar2 = sym.imp.strcmp((bright->window).sunlight, data.0804a05c);
         if (iVar2 == 0) {
             bright->ambassador = AMBASSADOR_REASON;
         } else {
@@ -2055,7 +2055,7 @@ void sym.Aeropause(BrightTypedefdPtr bright, int32_t argc, char **argv)
         bright->ambassador = AMBASSADOR_PURE;
     } else {
         (bright->window).sunlight = bright->morning->saved_argv[1];
-        iVar2 = sym.imp.strcmp((bright->window).sunlight, 0x804a05c);
+        iVar2 = sym.imp.strcmp((bright->window).sunlight, data.0804a05c);
         if (iVar2 == 0) {
             bright->ambassador = AMBASSADOR_REASON;
         } else {

